### PR TITLE
Add how methods are versioned to the help center

### PR DESCRIPTION
**Pull request by jhmjam**
This PR covers the work needed to add Steve's version policy information

### CODE REVIEW

**Review by sanjeevz3009 — APPROVED** [review]
Looks good =)

File: content/help_centre/version-methods.md
Comment (gibbardsteve) [inline content/help_centre/version-methods.md:9]: Could you make this line: The patch release version is incremented when backward compatible bugfixes are implemented.

It was a comment from Ian Dash when I shared the text with him.

File: content/help_centre/version-methods.md
Comment (JasonBellONS) [inline content/help_centre/version-methods.md:13]: Not sure on standard for accessibility, do hyperlinks have to match to the link itself or do they just need to give a good description of what they link to?

Comment (jhmjam) [reply content/help_centre/version-methods.md:13]: Not sure but this was written by Steve :)

File: content/help_centre/version-methods.md
Comment (jhmjam) [reply content/help_centre/version-methods.md:9]: Sure thing :)

File: content/help_centre/version-methods.md
Comment (JasonBellONS) [reply content/help_centre/version-methods.md:13]: Ah ok ill follow this up with steve separately, thanks

**Review by JasonBellONS — APPROVED** [review]
All looks good to me

**Review by gibbardsteve — APPROVED** [review]
Looks good, we do need to wait for Fleur before we merge this though.  We also need to make sure the PR for the creation of the release notes is actioned at the same time.

**Pull request merged**
